docs/docs/tutorials/using_dask.md

```diff
@@ -0,0 +1,234 @@
+
+# view the cluster options UI
+options = gateway.cluster_options()
+options
+```
+
+![Cluster Options UI](/img/cluster_options.png)
+
+Using the `Cluster Options` interface, we can specify the conda environment, the instance type, and any additional 
+environment variables we'll need. 
+
+:::warning
+It’s important that the environment used for your notebook matches the Dask worker environment!
+
+The Dask worker environment is specified in your deployment directory under `/image/dask-worker/environment.yaml`
+:::
+
+## Creating a Dask cluster
+
+```python
+# create a new cluster with our options
+cluster = gateway.new_cluster(options)
+# view the cluster UI
+cluster
+```
+
+![Creating a Gateway Cluster UI](/img/cluster_creation.png)
+
+We have the option to choose between `Manual Scaling` and `Adaptive Scaling`.
+
+If you know the resources that would be required for the computation, you can select `Manual Scaling` and 
+define a number of workers to spin up. These will remain in the cluster until it is shut down. 
+
+Alternatively, if you aren't sure how many workers you'll need, or if parts of your workflow require more workers
+than others, you can select `Adaptive Scaling`. Dask Gateway will automatically scale the number of workers
+(spinning up new workers or shutting down unused ones) depending on the computational burder. `Adaptive Scaling` is
+a safe way to prevent running out of memory, while not burning excess resources. 
+
+You may also notice a link to the Dask Dashboard in this interface. We'll discuss this in a later section. 
+
+## Viewing the Dask Client
+
+```python
+# get the client for the cluster
+client = cluster.get_client()
+# view the client UI
+client
+```
+
+![dask client ui](/img/dask_client.png)
+
+The `Dask Client` interface gives us a brief summary of everything we've set up so far. 
+
+## Now for the fun part - let's code with Dask! 
+
+We will perform some basic analysis on the well-known New York city yellow taxi dataset, a subset of which we have copied over to a Google Cloud Storage bucket `gs://nebari-public/yellow_taxi_NYC`. 
+
+:::note 
+This dataset is saved in parquet format, a column-oriented file format commonly used for large datasets saved in the cloud.
+:::
+
+
+To get started, we will load the data using dask dataframe. This will lazily load the dataset.
+
+```python
+import dask.dataframe as dd
+
+nyc_taxi_dask_df = dd.read_parquet(
+    path='gs://nebari-public/yellow_taxi_NYC/*/*.parquet', 
+    storage_options={'anon':True},
+)
+```
+
+From here we can start analyzing the data. First let's check the size of the overall dataset.
+
+```python
+dataset_size = nyc_taxi_dask_df.memory_usage(deep=True).compute().sum()
+dataset_size
+```
+
+**Output:**
+```shell
+32426244980
+```
+
+This corresponds to 32.43GB of data. Running this one-liner would be impossible on most single machines but running this on a dask cluster with 4 works, this can be calculated in under a minute.
+
+...
```

Comment:
@iameskild can you add whatever goes into this `...` and we can get this merged?

Reply:
Working on this as we speak :)